README.md

```diff
@@ -227,14 +297,15 @@ The HarbourBridge tool maps PostgreSQL types to Spanner types as follows:
 | `TIMESTAMP`        | `TIMESTAMP`            | t                             |
 | `TIMESTAMPTZ`      | `TIMESTAMP`            |                               |
 | `VARCHAR`          | `STRING(MAX)`          |                               |
-| `VARCHAR(N)`       | `STRING(N)`            |                               |
+| `VARCHAR(N)`       | `STRING(N)`            | c                             |
 | `ARRAY(`pgtype`)`  | `ARRAY(`spannertype`)` | if scalar type pgtype maps to spannertype |
 
 All other types map to `STRING(MAX)`. Some of the mappings in this table
 represent loss of precision (marked p), dropped autoincrement functionality
-(marked a), differences in treatment of timezones (marked t) and changes in
-storage size (marked s). We discuss each in turn, as well as other limits and
-notes on schema conversion.
+(marked a), differences in treatment of timezones (marked t), differences in
+treatment of fixed-length character types (marked c), and changes in storage
+size (marked s). We discuss each in turn, as well as other limits and notes on
+schema conversion.
```

Comment:
> We discuss each in turn, as well as other limits and notes on
schema conversion.

As is, this sentence is confusing. Did you mean:
> We discuss each in turn below, as well as other limits and notes on
schema conversion.